kafka-ballerina/caller.bal

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We do not need to change the license headers of the old files. This should contain the year the file is created. Please check the other places.